lib/configurators/aws/ec2discovery.go

```diff
@@ -37,13 +52,13 @@ mainSteps:
   name: downloadContent
   inputs:
     sourceType: "HTTP"
-    destinationPath: "/tmp/installTeleport.sh"
+    destinationPath: "/tmp/installTeleport-%s.sh"
```

Comment:
Mention somewhere in code that the script name is randomized to avoid guessing (and subsequent script "injections") ?